com.unity.multiplayer.mlapi/Editor/CodeGen/RuntimeAccessModifiersILPP.cs

```diff
@@ -89,6 +89,11 @@ private void ProcessNetworkManager(TypeDefinition typeDefinition)
                 {
                     fieldDefinition.IsPublic = true;
                 }
+                //IF TOOLS
+                if (fieldDefinition.Name == nameof(NetworkManager.__rpc_name_table))
+                {
+                    fieldDefinition.IsPublic = true;
+                }
```

Comment:
this was a workaround to support `2019.4` which we no longer support, so doesn't need to exist.
(and you just reminded me about a cleanup I'm supposed to do for the next release, no more `2019.4` workarounds, yay!)

Reply:
Okay, I'll remove that.